lib/active_merchant/billing/gateways/mercado_pago.rb

```diff
@@ -214,6 +215,10 @@ def add_taxes(post, options)
         end
       end
 
+      def add_payment_method_option_id(post, options)
```

Comment:
Because this new field is so closely related to using [the processing mode parameters for Mercado Pago](https://www.mercadopago.com.co/developers/en/guides/gateway/api/receiving-payments), this field would make more logical sense being added to the existing `add_processing_mode` method.

Reply:
https://github.com/activemerchant/active_merchant/pull/3635/commits/007ab496410a32775742ba9d92258cbc61952732

I've moved _ payment_method_option_id_ to the _add_processing_mode__ method and removed the _add_payment_method_option_id_ method